packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -236,6 +242,22 @@ export class VaLink {
       );
     }
 
+    if (external) {
+      return (
+        <Host>
+          <a
+            href={href}
+            class={linkClass}
```

Comment:
I noticed that USWDS has an [external link](https://designsystem.digital.gov/components/link/) example. Are any of their classes useful? `usa-link` or  `usa-link--external`?

Package usage: `@forward "usa-link";`

```html
  <a
    class="usa-link usa-link--external"
    rel="noreferrer"
    target="_blank"
    href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif"
    >external website</a
  >.
```

Reply:
Doesn't seem like this adds anything useful